app/jobs/ingest_job.rb

```diff
@@ -0,0 +1,17 @@
+class IngestJob < Hyrax::ApplicationJob
+  queue_as Hyrax.config.ingest_queue_name
+
+  after_perform do |job|
+    opts = job.arguments.reverse.detect { |x| x.is_a? Hash } || {}
+    wrapper = job.arguments.first
+    ContentNewVersionEventJob.perform_later(wrapper.file_set, wrapper.user) if opts[:notification]
```

Comment:
@atz What's happening here, as I currently understand it, is that we use `IngestJob` for two tasks: 1) to create a new work and 2) to upload a new version of a file to a work. We want notifications for both, but currently they're handled in different spots in the code, with only the latter triggered within the job. (The former is triggered by `Hyrax::Actors::BaseActor`.) The code we have now retains past behavior (triggering new version events when new versions are uploaded), so I'd be willing to punt this concern to a later date if it becomes problematic.